core/src/main/java/org/apache/iceberg/SnapshotManager.java

```diff
@@ -127,6 +127,32 @@ public ManageSnapshots rollbackTo(long snapshotId) {
     return setCurrentSnapshot(snapshotId);
   }
 
+  @Override
+  public Object updateEvent() {
+    if (targetSnapshotId == null) {
+      // NOOP operation, no snapshot created
+      return null;
+    }
+
+    switch (managerOperation) {
+      case ROLLBACK:
+        // rollback does not create a new snapshot
```

Comment:
It doesn't create a new snapshot, but should it send an event? Maybe as a follow-up?

Reply:
Yes. I'll create a followup for this. I presume this would be a specific Rollback event?

Reply:
Maybe a history modification event with a type that could be "rollback", "fast-forward", "cherry-pick", or "set-current"?